pkg/sdk/logging/model/output/vmware_log_intelligence.go

```diff
@@ -0,0 +1,151 @@
+// Copyright © 2024 Kube logging authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package output
+
+import (
+	"github.com/cisco-open/operator-tools/pkg/secret"
+	"github.com/kube-logging/logging-operator/pkg/sdk/logging/model/types"
+)
+
+// +name:"VMware Log Intelligence"
+// +weight:"200"
+type _hugoVMWareLogIntelligence interface{} //nolint:deadcode,unused
+
+// +docName:"VMware Log Intelligence output plugin for Fluentd"
+/*
+For details, see [https://github.com/vmware/fluent-plugin-vmware-log-intelligence](https://github.com/vmware/fluent-plugin-vmware-log-intelligence).
+## Example output configurations
+```yaml
+spec:
+  vmwarelogintelligence:
+    endpoint_url: https://data.upgrade.symphony-dev.com/le-mans/v1/streams/ingestion-pipeline-stream
+	verify_ssl: true
+	http_compress: false
+	headers:
+	  Content-Type: "application/json"
+	  Authorization: "Bearer 12345"
+	  structure: simple
+	buffer:
+	  chunk_limit_records: 300
+	  flush_interval: 3s
+	  retry_max_times: 3
+```
+*/
+type _docVMWareLogIntelligence interface{} //nolint:deadcode,unused
+
+// +name:"VMwareLogIntelligence"
+// +url:"https://github.com/vmware/fluent-plugin-vmware-log-intelligence/releases/tag/v2.0.8"
+// +version:"v2.0.8"
+// +description:"Send your logs to VMware Log Intelligence"
+// +status:"GA"
+type _metaVMWareLogIntelligence interface{} //nolint:deadcode,unused
+
+// +kubebuilder:object:generate=true
+// +docName:"VMwareLogIntelligence"
+type VMWareLogIntelligenceOutput struct {
+	// Log Intelligence endpoint to send logs to https://github.com/vmware/fluent-plugin-vmware-log-intelligence?tab=readme-ov-file#label-endpoint_url
+	EndpointURL string `json:"endpoint_url"`
+	// Verify SSL (default: true) https://github.com/vmware/fluent-plugin-vmware-log-intelligence?tab=readme-ov-file#label-verify_ssl
+	VerifySSL bool `json:"verify_ssl" plugin:"default:true"`
```

Comment:
If we add a default: true, then you won't be able to disable it. Please use a pointer to enable users disabling it.
```suggestion
	VerifySSL *bool `json:"verify_ssl" plugin:"default:true"`
```